File: Đề NĐ các năm/20-21/Bai XTB 2021/I91201/SDB.cpp
#include <iostream>
#include<fstream>

using namespace std;

int main()
{
    int n;
    int a[1000000];
    int dem =0;
    ifstream f ("SDB.inp");
    f>>n;
    for (int i=0;i<=n;i++)
        f>>a[i];
    ofstream g ("SDB.out");
    for (int i=0;i<=n;i++)
 if (a[1]%a[i]==1)
{
    dem =dem +0;
}
else
{
    dem =dem +1;
}
    f.close();
    g.close();
            return 0;
}

File: Đề NĐ các năm/20-21/Bai XTB 2021/I36202/SDB.pas
const fi='SDB.inp';
      fo='SDB.out';
var  n,max,i,j,dem,k:longint;
kt:boolean;
      f,g:text;
 a,c:array[1..1000000] of longint;

BEGIN
  assign(f,fi);
  reset(f);
  assign(g,fo);
  rewrite(g);
  readln(f,n);

  for i:=1 to n do
         readln(f,a[i]);
       k:=0;
  for i:=1 to n do
    begin
       dem:=0;
    for j:=1 to n do
        if a[j]=a[i] then inc(dem);
        if dem=1  then
        begin
          inc(k);
          c[k]:=a[i];
        end;
     end;
     writeln(g,k);
     for i:=1 to k do writeln(g,c[i]);
       close(f); close(g);
END.
